accurest-core/src/test/groovy/io/codearte/accurest/dsl/WireMockGroovyDslSpec.groovy

```diff
@@ -183,6 +183,7 @@ class WireMockGroovyDslSpec extends Specification implements WireMockStubVerifie
 		when:
 			String wireMockStub = new WireMockStubStrategy("Test", new Contract(null, false, 0, null), groovyDsl).toWireMockClientStub()
 		then:
+			//TODO: That assertion fails on some environments due to 2 ways how body array is created - with 'created' at the beginning or 'id'
```

Comment:
Maybe instead of adding a comment this assertion could be modified to compare the jsons wtih problematic elements removed on both sides?